### PR TITLE
OLS-426: Add step to run lint check

### PR DESCRIPTION
**Pull request by sauagarwa**

### CODE REVIEW

File: ci-operator/config/openshift/lightspeed-console/openshift-lightspeed-console-main.yaml [hunk 1]
```diff
@@ -32,6 +32,17 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
+tests:
+- as: lint-aws-plugin-template
```
Comment (bparees) [inline ci-operator/config/openshift/lightspeed-console/openshift-lightspeed-console-main.yaml:36]:
```suggestion
- as: lint
```

unless you expect to have multiple lint jobs

**Review by bparees — CHANGES_REQUESTED** [review]
you also need to run "make update" to generate the job from the config

**Comment by sauagarwa** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-console-main-images

**Comment by sauagarwa** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-console-main-images

**Comment by sauagarwa** [comment]
/pj-rehearse ci/rehearse/openshift/lightspeed-console/main/images

**Comment by sauagarwa** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-console-main-images

**Comment by sauagarwa** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-console-main-images

**Comment by sauagarwa** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-console-main-images

**Comment by sauagarwa** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-console-main-images

**Comment by sauagarwa** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-console-main-images

**Comment by bparees** [comment]
/pj-rehearse ack

@sauagarwa for future reference don't forget you need to ack that the rehearsals have been run and passed (to ensure we don't merge a CI change that breaks the CI jobs and prevents PRs from merging)

**Pull request merged**